src/sections/Skills/Skills.svelte

```diff
@@ -26,6 +26,10 @@
         titleLabel="Project Management"
         paragraphContent="Planning and Organization, Risk Management, Leadership, Problem-solving, JIRA, Scrum."
       />
+      <Skill
+        titleLabel="Hipster languages"
+        paragraphContent="Finding hipster programming langauges and packages so I can say I liked them before they were cool"
```

Comment:
Phrase has no dot at the end. Not a valid English phrase.

Reply:
That's it. I quit!